docs/guides/classic-ui/index.md

```diff
@@ -0,0 +1,25 @@
+---
+myst:
+  html_meta:
+    "description": "User Guide for Plone 6 Classic"
```

Comment:
Replace any and all instances of "Classic" with "Classic UI".

```suggestion
    "description": "User Guide for Plone 6 Classic UI"
```

Reply:
Done